catkit2/services/simple_simulator/simple_simulator.py

```diff
@@ -20,7 +20,7 @@ def open(self):
 
         self.images = self.make_data_stream('images', 'float64', self.model.focal_grid.shape, 20)
 
-        self.update_atmosphere()
+        self.update_atmosphere
```

Comment:
Yes it is. Before deleting the parenthesis, I had an error ```TypeError: update_atmosphere() takes 1 positional argument but 2 were given```. It is the only solution I found to solve this problem.

Reply:
I find that weird. Let me have a look at this and I'll come back to you, because currently I can't find an explanation for this behavior.

Reply:
@ehpor any clue what might be happening here? @erinpougheon could you post the error message you're getting for this here please?

Reply:
![image (13)](https://github.com/user-attachments/assets/2aa709ef-abdf-4a06-95a1-add065bfb85b)

Reply:
Hmmm I think you're avoiding the error, but I don't think the atmosphere works either. It just doesn't do anything. 

If you look at the simulator.py code here, https://github.com/spacetelescope/catkit2/blob/a7c17df19636676151d965e67d0085367ba14769/catkit2/simulator/simulator.py#L131C13-L131C41, it calls the callback function with time as an argument. So it will call ``self.update_atmosphere(t)``, that has 2 arguments (self, and t). You would need to define ``def update_atmosphere(self, t)`` or something like that.

Also, looking at the code, I feel the ``simple_simulator`` on develop is bugged.